### PR TITLE
Extreme networks vsp 7254xsq

### PR DESCRIPTION
**Pull request by WagoL**
added Extreme networks vsp-7254xsq device type. 
added PSU module for in the vsp-7254xsq.

### CODE REVIEW

File: module-types/Extreme Networks/EC72051AF-E6.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+manufacturer: Extreme Networks
+model: EC72051AF-E6
```
Comment (danner26) [inline module-types/Extreme Networks/EC72051AF-E6.yaml:3]: I am having a hard time finding `EC72051AF-E6`, but I have found `EC7205A1F-E6`

Is the proper model name `EC7205A1F-E6`?

File: device-types/Extreme Networks/VSP-7254XSQ.yaml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+manufacturer: Extreme Networks
+model: VSP-7254XSQ
+slug: extreme-networks-vsp-7254xsq
+part_number: VSP-7254XSQ
+u_height: 1
+is_full_depth: true
+weight: 7250
+weight_unit: g
+airflow: front-to-rear
+subdevice_role: parent
```
Comment (danner26) [inline device-types/Extreme Networks/VSP-7254XSQ.yaml:11]: This is generally used for chassis which house other devices, this is not meant for devices with modules. Is this actually needed, and if so why are there no `device-bays`?

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Extreme Networks/VSP-7254XSQ.yaml (hunk 2 above)
Comment (WagoL) [reply device-types/Extreme Networks/VSP-7254XSQ.yaml]: removed the parent role.

File: module-types/Extreme Networks/EC72051AF-E6.yaml (hunk 1 above)
Comment (WagoL) [reply module-types/Extreme Networks/EC72051AF-E6.yaml]: This is indeed a typo. Made the change.

**Pull request merged**